sdk/iot/hub/samples/README.md

```diff
@@ -0,0 +1,76 @@
+---
+page_type: sample
+languages:
+  - c
+products:
+  - azure
+  - azure-iot
+  - azure-iot-hub
+urlFragment: iot-hub-samples
+---
+
+# Azure IoT Hub Sample
+
+This document explains samples for the Azure Embedded C SDK IoT Hub Client and how to use them.
+
+## Key concepts
+
+Key concepts are explained in detail [here][SDK_README_KEY_CONCEPTS].
+
+## Samples for Azure IoT Hub Client APIs
+
+This document describes how to use samples and what is done in each sample.
+
+## Getting started
+
+Getting started explained in detail [here][SDK_README_GETTING_STARTED].
+
+## Examples
+
+The following section documents various examples. All of them use the [Eclipse Paho MQTT C client][Eclipse_Paho].
+
+### [IoT Hub C2D][c2d_sample]
+Receive and view incoming C2D messages using the IoT Hub client.
+
+### [IoT Hub Methods][methods_sample]
+Invoke methods from the cloud. The sample supports a method named "double"
+which will return back to you the value sent as the payload. The payload must be of the form:
+```json
+{
+  "value": 10
+}
+```
+where 10 can be substituted for any number between `UINT64_T_MIN / 2` and `UINT64_T_MAX / 2`.
```

Comment:
Isn't UINT64_T_MIN just 0? Was this meant to be INT64_T? Also, consider using the names from `limits.h`
https://www.tutorialspoint.com/c_standard_library/limits_h.htm

I don't get much search hit for `UINT64_T_MAX`.

```suggestion
where 10 can be substituted for any number between `0 and `ULONG_MAX / 2`.
```

Reply:
yes it should be `INT32` thank you